app/configurator/map/map-chart-options.tsx

```diff
@@ -133,8 +133,11 @@ export const AreaLayerSettings = memo(
       [metaData.measures]
     );
 
-    const numberOfGeoShapes = (dimension.geoShapes as any).topology.objects
-      .shapes.geometries.length as number;
+    const numberOfGeoShapes = (
+      dimension
+        ? (dimension.geoShapes as any).topology.objects.shapes.geometries.length
```

Comment:
I wonder why we have to resort to any here ?

Reply:
Good point, we don't have to :)